app/src/main/java/com/internship/move/model/appModule.kt

```diff
@@ -45,6 +49,7 @@ fun provideRetrofit(moshi: Moshi, client: OkHttpClient) = Retrofit.Builder()
 
 fun provideMoshi(): Moshi = Moshi.Builder().build()
 
-fun provideHttpClient(): OkHttpClient = OkHttpClient.Builder().addInterceptor(HttpLoggingInterceptor()).build()
+fun provideHttpClient(): OkHttpClient =
+    OkHttpClient.Builder().addNetworkInterceptor(HttpLoggingInterceptor().setLevel(HttpLoggingInterceptor.Level.BODY)).build()
```

Comment:
If you instantiate it like this, the interceptor will output console logs even on production release builds
Implement a way to add the LoggingInterceptor only on debug builds